src/omneo/resources/profiles/index.ts

```diff
@@ -444,6 +445,15 @@ export default class Profiles extends Resource {
     })
   }
 
+  async deleteProfileDate (profileID: string, dateId: string): Promise<Array<ProfileDatesAttribute>> {
```

Comment:
Promise<ProfileDatesAttribute[]>
just to be consistent with the rest of the repo